### ISSUE TITLE
A single configuration file for all settings

### ISSUE THREAD

**Issue by probstlukas** [FliegendeWurst/KIT-ILIAS-downloader#23]
Files such as .iliasignore, .iliaslogin, course_names.toml and the flags/options that can be set via the command itself should be combined in a single configuration file such as downloader.conf.

**Comment by floriensk** [comment]
I would add that the `.iliaslogin` is a bit unhandy as it has to be present in the target folder, which might be different for each course download. It would be, I think, wiser to search for it (or any config file) in the directory where the executable itself is.

**Comment by FliegendeWurst** [comment]
Good point. However, the `.iliasignore` configuration probably needs to be different per download target.. maybe it would be best to have a central config file (combining login + course names) and separate configs for each download directory (.iliasignore).

**Comment by floriensk** [comment]
That's a good point, I'd love to see this in an upcoming update. :)
